src/main.rs

```diff
@@ -160,11 +173,11 @@ fn handle_device(config: &Config, device: &Device, memtotal_mb: f64) -> Result<b
                  device.memory_limit_mb);
         return Ok(false);
     }
-    
+
     let disksize = (device.zram_fraction * memtotal_mb) as u64 * 1024 * 1024;
     let service_name = format!("swap-create@{}.service", device.name);
-    println!("Creating {} for /dev/{} ({}MB)",
-             service_name, device.name, disksize / 1024 / 1024);
+    println!("Creating {} for {}dev/{} ({}MB)",
+             service_name, config.root, device.name, disksize / 1024 / 1024);
```

Comment:
I don't think prefixing /dev with the root path makes sense. We are not creating an actual device file anywhere underneath config.root.